### DISCUSSION TITLE
Clearer information on reverse proxy with AIO interface

### DISCUSSION THREAD

**Discussion by brendan-pike** [nextcloud/all-in-one#4353] [Questions] [answered]
I've configured AIO behind an Nginx with a valid certificate and everything works well except I can't reach the AIO interface using Firefox on port 8443 as I get 
`Did Not Connect: Potential Security Issue`
`The certificate does not come from a trusted source.`
I've followed https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#5-optional-get-a-valid-certificate-for-the-aio-interface but do not understand how to adapt point 5 to a Nginx conf?
Can this be better explained please.

**Comment by szaimen** [comment, marked as answer]
Hi, port 808 should work always: https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#4-open-the-aio-interface

**Comment by brendan-pike** [comment]
Thank you 8080 does "work" but modern browsers refuse to let you continue since the certificate is invalid.
Chrome says 
`NET::ERR_CERT_AUTHORITY_INVALID`
and has no option to continue or ignore, Firefox is the same.

**Reply by szaimen** [reply]
did you try `https://server.ip.address:8080` ?

**Comment by brendan-pike** [comment]
No I hadn't. Okay using the IP will allow me manually accept the risk and continue, I then had to go back to the TLD /settings/admin/overview in order to use the AIO Login button, which then redirected to the TLD but then I changed the URL manually to IP and was able to login.  So rather convoluted steps but I got there thanks to your IP tip.  Including the IP tip would be very helpful to nginx users. Even better would be steps on how to get a valid cert issued.
